materials/module-1/worksheets/2-peer-to-peer-file-sharing-on-a-mesh.md

```diff
@@ -0,0 +1,80 @@
+Peer-to-peer file sharing on a mesh
+===================================
+
+## Connecting to your Raspberry Pi
+
+1. Verify that the Raspberry Pi has its WiFi adapter(s) proper connected SD card is in place. Power on the device and wait for a solid green LED with a flashing red LED.
+
+1. Connect your computer's WiFi Client to the Raspberry Pi's WiFi Access Point:
+
+        SSID:       <hostname>
+        Password:   password
+
+1. Once connected to the Access Point from your computer, you can connect to it with SSH password `root`:
+
+        $ ssh root@<hostname>.local
```

Comment:
add Password?

Reply:
He states the password in the line above

> you can connect to it with SSH password `root`:

But I agree it may need to be spelled out like you spell out the SSID, people in the workshop were getting a bit confused about that

Reply:
There were a couple points of confusing here, including PuTTY using a different format. I will address them together.

Reply:
FYI: putty you enter the host name without the user name and you get prompted for the user name (similar to  telnet)